sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLPoolDAO.java

```diff
@@ -371,6 +372,23 @@ public void setWatcherDAO(WatcherStore watcherDAO) {
   public void setCascadeType(CascadeType cascadeType) {
     this.cascadeType = cascadeType;
   }
+  
+  public void setAutoGenerateIdentificationBarcodes(boolean autoGenerateIdentificationBarcodes) {
+    this.autoGenerateIdentificationBarcodes = autoGenerateIdentificationBarcodes;
+  }
+  
+  public boolean getAutoGenerateIdentificationBarcodes() {
+    return autoGenerateIdentificationBarcodes;
+  }
+  
+  /**
+   * Generates a unique barcode. Note that the barcode will change if the Platform is changed.
+   * @param pool
+   */
+  public void autoGenerateIdBarcode(Pool pool) {
```

Comment:
more generic warnings.  I notice this is never used in our code either.  I'll make a note to review.